provider/aws/formation/resource/mariadb.json.tmpl

```diff
@@ -1,15 +1,14 @@
 {
   "AWSTemplateFormatVersion": "2010-09-09",
   "Conditions": {
-    "BlankEncrypted": { "Fn::Equals": [ { "Ref": "Encrypted" }, "" ] },
```

Comment:
why remove this one? not used?

Reply:
It was not being used but I included it again because setting the value as `false` will cause the replacement.